src/App.tsx

```diff
@@ -1,14 +1,14 @@
-import { useEffect, useState } from "react";
+import React, { useEffect, useState } from "react";
 import { HashRouter, Route, Routes } from "react-router-dom";
 import { Footer } from "@/components/Footer/Footer";
 import SideSheet from "@/components/common/SideSheet/SideSheet";
 import Learn from "@/components/Learn/Learn";
-import Root from "@/model/root";
+import QuizView from "@/components/Quiz/QuizView";
 import "./App.scss";
-
-const ROOT_DATA = Root.get();
+import creatureRepository from "./model/creature/creature-repository";
 
 function App() {
+    const [isQuiz, setIsQuiz] = useState(false);
```

Comment:
Please use a different route for quiz.
I will create one for learn. Home page will be 2 buttons to chose from Learn and Quiz.